tests/D2L.CodeStyle.Analyzers.Test/Specs/ReflectionSerializerAnalyzer.cs

```diff
@@ -80,7 +80,7 @@ public sealed class ParameterCannotBeDeserialized_BecauseNameMismatch {
 				X = value;
 			}
 			public int X { get; }
-        }
+		}

```

Comment:
Ignore the whitespace changes.  Editor was throwing in spaces without me realizing it.  This fixes them to tabs